### ISSUE TITLE
Exporting/Importing Labels & Comments

### ISSUE THREAD

**Issue by winkerbie** [x64dbg/x64dbg#1471]
Is it possible to import/export labels, comments and breakpoints for DLLs and EXEs if they were renamed, different versions, different directory, or DLLs loaded by different EXEs?

Just checking, cant seem to find a option in the menus

**Comment by mrexodia** [comment]
Yes, you can import/export the database from the file menu. You will have to manually change the module.

**Comment by ThunderCls** [comment]
https://github.com/ThunderCls/xLCB
